akka-stream-tests/src/test/scala/akka/stream/impl/InterpreterSpec.scala

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.impl
+
+import akka.stream.Supervision._
+import akka.stream.testkit.AkkaSpec
+import akka.stream._
+import akka.stream.scaladsl._
+import akka.stream.testkit.StreamTestKit._
+import akka.stream.impl.fusing.ActorInterpreter
+
+class InterpreterSpec extends AkkaSpec {
```

Comment:
This is not related to fusing at all; it should probably be called ActorInterpreterSpec.